src/spatial/math.js

```diff
@@ -703,9 +712,9 @@ Crafty.math.Vector2D = (function () {
      * If dy is omitted, dx is used for both axes.
      *
      * @public
-     * @sign public {Vector2D} translate(Number[, Number]);
-     * @param {Number} dx
-     * @param {Number} [dy]
+     * @sign public {Vector2D} translate(Number x[, Number y]);
```

Comment:
I think that's supposed to indicate an optional parameter, but since it's not used elsewhere in Crafty let's just remove it.